spec/takeaway_spec.rb

```diff
@@ -0,0 +1,54 @@
+require 'takeaway'
+
+describe Takeaway do
+
+  subject(:takeaway) {described_class.new(menu)}
+  let(:menu) {double(:menu, :menu_list => {"chicken" => 15}) }
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)